### PR TITLE
Add GitHub Actions

### PR DESCRIPTION
**Pull request by eumiro**
Let's see whether this is a good replacement for the current `.travis.yml` file…

### CODE REVIEW

**Comment by eumiro** [comment]
Since the `master` branch does not contain GHA until now, we cannot see the results of the GHA CI run here.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+name: CI
+
+on:
+  push:
+    branches: ["master"]
```
Comment (xflr6) [inline .github/workflows/ci.yml:5]: Better to continue to run the CI on every pushed branch (provides an easy way to test things before creating a PR).

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+          sudo apt-get install graphviz
+      - name: "Install Python dependencies"
+        run: |
+          set -xe
+          python -VV
+          python -m site
+          python -m pip install --upgrade pip setuptools wheel
+          python -m pip install --upgrade virtualenv tox tox-gh-actions
+      - name: "Run tox targets for ${{ matrix.python-version }}"
+        run: "python -m tox"
```
Comment (xflr6) [inline .github/workflows/ci.yml:36]: Not sure if we should start using `tox` on CI, IIRC there are some problems to get coverage (paths) properly.

**Comment by xflr6** [comment]
Thanks, left some comments.,

File: .github/workflows/ci.yml (hunk 1 above)
Comment (eumiro) [reply .github/workflows/ci.yml:5]: okay, updated

**Comment by eumiro** [comment]
BTW, the results of this PR can be found at https://github.com/eumiro/graphviz/actions

File: .github/workflows/ci.yml (hunk 2 above)
Comment (eumiro) [reply .github/workflows/ci.yml:31]: Now I rewrote it a little bit and the coverage seems to work directly within `tox`.

**Comment by eumiro** [comment]
Added a `README.rst` badge for the GHA CI. Please squash the commits afterwards.

**Comment by xflr6** [comment]
Sorry for the delay.

Still prefer not using `tox` in the CI and want to continue collecting `coverage` from all jobs (to cover for compat code branches).

Superseded by a0668eaab56a1ae56da8b826795bf54d7a405502 and 78885a833524e48cfcbe08694dc151aaa2c2705a, closing, thanks for the PR.

**Pull request closed without merging**